benchopt/utils/dependencies_mixin.py

```diff
@@ -170,19 +179,19 @@ def collect(cls, env_name=None, force=False):
         conda_reqs, shell_install_scripts, post_install_hooks = [], [], []
```

Comment:
Actually, the we should also check in this method that the `install_cmd` is valid. (that's the reason why the test fail in the CI).